psimol/main.py

```diff
@@ -443,7 +443,57 @@ def from_mol(cls, file_path: str) -> Molecule:
         Args:
             file_path (str): Path to the .mol file
         """
-        pass #TODO
+        
+        with open(file_path, 'r') as file:
+            lines = file.readlines()
+
+        # first line of file may or may not contain molecule name
+        name = lines[0].strip()
+        # fourth line contains atom and bond count
+        counts = lines[3].strip()
+
+        # fields in .mol files are fixed-width, rather than delineated by whitespace
+        atom_cnt = int(counts[0:3])
+        bond_cnt = int(counts[3:6])
+
+        atom_block = lines[4          : 4+atom_cnt]
+        bond_block = lines[4+atom_cnt : 4+atom_cnt+bond_cnt]
+        print(bond_cnt, len(bond_block))
```

Comment:
```suggestion
        logging.debug(bond_cnt, len(bond_block))
```
Guess it's rather for debugging.